examples/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "examples",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
```

Comment:
Should add a 
```json
"private": true
```
attribute to ensure this is never accidentally published to NPM.